src/sdgio-edit.owl

```diff
@@ -3327,13 +3327,13 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020304> "reform policies and institutions"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00020304>)
 
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020305> (increase agri-food research productivity and of innovation in the agri-food system)
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020305> (increase agri-food research productivity and innovation in the agri-food system)
```

Comment:
Change to:
"increase research productivity and innovation in the agri-food system"

Reply:
done, thanks